src/main/charts/confluence/README.md

```diff
@@ -33,29 +33,23 @@ Kubernetes: `>=1.19.x-0`
 | confluence.additionalBundledPlugins | list | `[]` | Specifies a list of additional Confluence plugins that should be added to the  Confluence container. Note plugins installed via this method will appear as  bundled plugins rather than user plugins. An alternative to this method  is to install the plugins via "Manage Apps" in the product system  administration UI. These should be specified in the same manner as the  'additionalLibraries' property. |
 | confluence.additionalEnvironmentVariables | list | `[]` | Defines any additional environment variables to be passed to the Confluence  container. See https://hub.docker.com/r/atlassian/confluence-server for  supported variables. |
 | confluence.additionalJvmArgs | list | `["-XX:ActiveProcessorCount=2"]` | Specifies a list of additional arguments that can be passed to the Confluence JVM, e.g.  system properties. |
-| confluence.additionalJvmArgs[0] | string | `"-XX:ActiveProcessorCount=2"` |  for 'container.requests.cpu'. https://docs.oracle.com/en/java/javase/11/tools/java.html#GUID-3B1CE181-CD30-4178-9602-230B800D4FAE |
```

Comment:
Were these deleted manually? It is good catch, we should include just the expanded values instead of the full  objects but I wonder why my pre-commit didn't catch this.

Reply:
I didn't change them manually - All caught by pre-commit.

Reply:
@nghazali please leave the conversation open if there are still other reviewers, the comment might be useful for them.